configs/.zshrc

```diff
@@ -28,12 +28,12 @@ bash /home/astronaut/.astroarch/scripts/aa_motd.sh
 
 function use-astro-bleeding-edge()
 {
-    sudo pacman -Sy && yes | sudo pacman -S kstars-git libindi-git indi-3rdparty-drivers-git indi-3rdparty-libs-git
+    sudo pacman -Sy && yes | LC_ALL=en_US.UTF-8 sudo pacman -S kstars-git libindi-git indi-3rdparty-drivers-git indi-3rdparty-libs-git
```

Comment:
@sc74 this change is very important and should make it to upstream (main branch) asap. Can you issue a PR to main with this change only so we will ship it to users? Or I may port it to main if you want. Let me know